kube/api-service.yaml

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2023 KernelCI Authors.
+# Author: Sanskar Bhushan <sbdtu5498@gmail.com>
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: kernelci-api
+spec:
+  selector:
+    app: kernelci-api
+  ports:
+    - protocol: TCP
+      port: 8000
+      targetPort: 8000
+  type: ClusterIP
```

Comment:
I guess this can be eliminated as the default service type is `ClusterIP`.

Reply:
Yes, this can be. I have included it to increase clarity.